index.html

```diff
@@ -13,6 +13,18 @@
 	<header>
 		<h1>JS Quiz App</h1>
 	</header>
+	<div class="content">
+		<div class="progress">
+			<div class="progress-track"></div>
+			<div id="step1" class="progress-step">
+			</div>
+			<div id="step2" class="progress-step">
+			</div>
+			<div id="step3" class="progress-step">
+			</div>
+			<div id="step4" class="progress-step">
+			</div>
```

Comment:
Ponumeruj id stepów od 0. Będzie wtedy tak samo jak current question a to nam się przyda